### ISSUE TITLE
`rumdl init` suggests a configuration that rumdl does not recognize?

### ISSUE THREAD

**Issue by telemachus** [rvben/rumdl#106]
After running `rumdl init`, I get a TOML configuration that includes these lines (commented out initially).

```markdown
# [MD044]
# names = ["rumdl", "Markdown", "GitHub"]  # Proper names that should be capitalized correctly
# code_blocks_excluded = true  # Exclude code blocks from proper name check
```

I uncommented the three lines, but then `rumdl check .` gives me this error message.

```shell
[config warning] Unknown option for rule MD044: code-blocks-excluded
```

In case it matters, here's some more information.

+ rumdl 0.0.159
+ rustc 1.90.0 (1159e78c4 2025-09-14)
+ macOS 26.0.1

**Comment by rvben** [comment]
Good day, @telemachus.

Thanks for making me aware. I've updated the config that comes out of the 'rumdl init' command. Also, there were no tests to verify this, and I've added that. This makes sure we don't hit issues like this in the future.

These fixes should land in 0.0.160.

**Comment by telemachus** [comment]
Thanks for the quick reply and thanks for rumdl. I love that I can get uniform markdown formatting easily.
